src/index.md

```diff
@@ -1693,6 +1693,71 @@ reverse effect of earlier changeset
     merged result is left uncommitted.
 ~~~
 
+# JDK Release process
+
+::: {.box}
+[Quick Links]{.boxheader}
+
+* [JEP 3: JDK Release Process](https://openjdk.java.net/jeps/3)
+:::
+
+The JDK project has a well defined release process. [JEP 3](https://openjdk.java.net/jeps/3) describes this process in detail. This section tries to clarify some topics that often cause questions.
+
+## Release cycle
+
+The release cycle starts when development of a new release begins and ends when that release is delivered to the public. The current release cadence is six months. This means that every six months we start development of a new release, and every six months a new release is delivered. However, this doesn't mean that each release cycle is six months. As described below, the total development time for a release (the release cycle) is actually nine months. Obviously this in turn doesn't mean that all features are developed in nine months. Most features are developed for much longer time than that, and goes through long time development in other project repositories, and through a series of preview and experimental stages. But any feature that is to be included in a specific release has a specific window of nine months to integrate the code into mainline and fix all the remaining bugs.
+
+It may be tempting to integrate a new feature near the end of a release cycle, to get more time to fix all those last bugs before integration. Please don't. If you are getting close to the end of a release and you still just have one more bug to fix, please defer your feature to the next release. It's only six months out. Not only will this vouch for your new feature to be more stable on release, you will also help keeping the JDK as a whole more stable by allowing others to find and fix bugs in their new code that might come as a result of your changes.
+
+Integrating early in a release is preferable, but all new features can't be integrated at the same time. If many large changes enters the repository at the same time it will be more difficult to determine which change that caused all the new bugs. If you're about to integrate a larger change you must therefore communicate this on the relevant [mailing lists](#mailing-lists) to synchronize with other projects that may also be planning to integrate something soon.
+
+## Milestones and phases
+
+Throughout the release there are a number of milestones and phases that define where in the release cycle we are.
+
+[**The start of a release**]{#release-start}
+:    Since development is always ongoing in the mainline repository ([openjdk/jdk](https://github.com/openjdk/jdk)), the start of a new release can be said to begin when the former release is forked of from mainline. After the start of the release follows six months of development to implement and integrate all the cool stuff that will go into the next release. After these six months rampdown begins.
```

Comment:
wording: probably "forked from the mainline" at the end of the first sentence.

Reply:
Fixed